fe/fe-core/src/main/java/com/starrocks/catalog/BrokerMgr.java

```diff
@@ -51,7 +51,7 @@
  */
 public class BrokerMgr {
     public static final ImmutableList<String> BROKER_PROC_NODE_TITLE_NAMES = new ImmutableList.Builder<String>()
-            .add("Name").add("Host").add("Port").add("Alive")
```

Comment:
I'm afraid you can't change the title.
Because some old client code depend on this name.
If you change the name, their code will produce error.

Reply:
A previous PR changed the ip to host, and then did find the problem you just mentioned, so this PR is to change the host back to ip